src/main/java/de/chojo/repbot/commands/TopWeek.java

```diff
@@ -0,0 +1,49 @@
+package de.chojo.repbot.commands;
+
+import de.chojo.jdautil.command.SimpleCommand;
+import de.chojo.jdautil.localization.Localizer;
+import de.chojo.jdautil.localization.util.LocalizedEmbedBuilder;
+import de.chojo.jdautil.localization.util.Replacement;
+import de.chojo.jdautil.wrapper.SlashCommandContext;
+import de.chojo.repbot.data.ReputationData;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+
+import javax.sql.DataSource;
+import java.awt.Color;
+import java.util.stream.Collectors;
+
+import static de.chojo.repbot.commands.Top.buildTop;
+
+public class TopWeek extends SimpleCommand {
+    private static final int TOP_PAGE_SIZE = 10;
+    private final ReputationData reputationData;
+    private final Localizer loc;
+
+    public TopWeek(DataSource dataSource, Localizer localizer) {
+        super("topweek",
+                new String[]{"reptop"},
```

Comment:
Same here, does this result in a conflict?

Reply:
This is only used for legacy text commands
```suggestion
                null,
```